src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.view;
+
+import racingcar.model.Car;
+
+import java.util.List;
+
+public class OutputView {
+    private static final String RESULT_HEADER = "실행 결과";
+    private static final String WINNER_MESSAGE = "최종 우승자 : %s%n";
+
+    public void displayHeader() {
+        System.out.println(RESULT_HEADER);
+    }
+
+    public void displayWinners(List<Car> winners) {
+        String winnerNames = winners.stream()
+                .map(Car::getName)
+                .reduce((a, b) -> a + ", " + b)
+                .orElse("");
```

Comment:
```suggestion
        String winnerNames = String.join(", ", winnerNames);
```
String의 join메서드를 활용하면 코드량을 줄일 수 있을 것 같습니다!

Reply:
String 클래스의 메서드를 많이 알고 있지 못하다 보니 공부하고 있던 스트림을 사용해 코드를 짰던 것 같아요 메서드 활용을 늘려 피드백 주신 대로 코드를 간결하게 할 수 있다니...... 역시 아는 만큼, 공부한 만큼 보이나 봅니다 감사해요!